cypress/integration/sidebar.spec.js

```diff
@@ -0,0 +1,21 @@
+const autoRecord = require('cypress-autorecord');
+
+describe('The Sidebar', () => {
+  autoRecord();
+
+  before(() => {
+    cy.login()
+  })
+
+  it('click in the user avatar opens sidebar', function () {
+    cy.get('[alt="USER"]').click()
+
+    cy.get('.sidebar__nav-menu-active').should('exist');
+  })
+
+  it('click in the user avatar in the sidebar closes the sidebar', function () {
+    cy.get('[alt="USER"]').click()
+
+    cy.get('.sidebar__nav-menu-active').should('not.exist');
+  })
```

Comment:
@rafayet-monon  How about adding `describe` to make the test cases more clear, maybe something like this? 🤔 

```suggestion
  describe('when click on the user avatar', () => {
    it('opens the sidebar', () => {
      cy.get('[alt="USER"]').click();
  
      cy.get('.sidebar__nav-menu-active').should('exist');
    });
  });

  describe('when the sidebar is already open and click on the user avatar', () => {
    it('closes the sidebar', () => {
      cy.get('[alt="USER"]').click();
  
      cy.get('.sidebar__nav-menu-active').should('not.exist');
    });
  });
```